src/lib/develco.ts

```diff
@@ -170,4 +171,61 @@ export const develcoModernExtend = {
             valueIgnore: [0xffff, -0x8000],
             ...args,
         }),
+    current_summation: () =>
```

Comment:
Could you refactor it to e.g. `numeric()`? Example can be found [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/ff9ad52aa88c7b275599ad568e66af9720aa5c0a/src/lib/develco.ts#L66)

Reply:
current_summation is a write only entity, so I do not see any idea in converting that to `numeric()` as that seems like read/write?
But perhaps the pulse_configuration can be changed.. I'll look into it.

Reply:
We can add `access: 'SET'` option (also has to be added to `numeric`)

Reply:
That is a bit our of my league at the moment.. Also looked into converting the pulse_config to something more generic (like numeric), but seems a bit overcomplicated for this pr (at the moment)

Reply:
Can you try with:

```
        numeric({
            name: 'pulse_configuration',
            cluster: 'seMetering',
            attribute: 'develcoPulseConfiguration',
            description: 'Pulses per kwh. Default 1000 imp/kWh. Range 0 to 65535',
            access: 'ALL',
            unit: '%',
            valueMin: 0,
            valueMax: 65535,
        }),
```

Reply:
I have added the suggestion. But am unable to test if it works at the moment (doesn't have access to the house where my device is located for at least a week)